cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -713,3 +716,66 @@ def test_custom_button_open_url_service_obj(objects, button_group):
         1550002
     """
     pass
+
+
+@pytest.fixture(params=["Service", "Provider"], scope="module")
+def unassigned_btn_setup(request, appliance, provider, service):
+    if request.param == "Service":
+        obj = MyService(appliance, name=service.name)
+        destinations = [ViaUI, ViaSSUI]
+    else:
+        # only service is different than other custom button object so selecting one i.e. provider
+        obj = provider
+        destinations = [ViaUI]
+
+    gp = appliance.collections.button_groups.instantiate(
+        text="[Unassigned Buttons]", hover="Unassigned buttons", type=request.param
+    )
+    yield obj, gp, destinations
+
+
+@pytest.mark.provider([VMwareProvider], override=True, scope="module", selector=ONE)
+def test_custom_button_unassigned_behavior_objs(
+    appliance, setup_provider, unassigned_btn_setup, request
+):
+    """ Test unassigned custom button behavior
+
+    Note: Service unassigned custom button should display on SSUI but not OPS UI.
+    For other than service objects also follows same behaviour i.e. not display on OPS UI.
+
+    Polarion:
+        assignee: ndhandre
+        initialEstimate: 1/6h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.8
+        casecomponent: CustomButton
+        testSteps:
+            1. Create unassigned custom button on service and one other custom button object.
+            2. Check destinations OPS UI should not display unassigned button but SSUI should.
```

Comment:
Steps say "OPS UI should not display unassigned button" but you are asserting button.exists

Reply:
@sshveta 
`L771` point `button.exists` means check button created properly or not it's not checking on the destination page.
`L781` is actual check where we are checking OPS and SSUI difference.